### ISSUE TITLE
Cannot connect to saved Postgres connections in 3.8.1

### ISSUE THREAD

**Issue by BertHartm** [serge-rider/dbeaver#1049]
I've been using dbeaver to connect to postgres databases for the past few versions.

Since I've upgrade to 3.8.1, I can no longer connect to those databases. I uninstalled and removed my `~/.dbeaver` folder, and recreated them, and that worked for the initial connection, but then the issue returned, and removing and recreating the connection did not work.

When I open the connection, it appears to work briefly, but running a query complains "Not connected to database" and nothing I do seems to fix that.

Additionally, attempting to close dbeaver, or delete the connection once I've attempted to connect causes dbeaver to hang and I'm forced to kill it.

I'm using dbeaver-ce 3.8.1 on ubuntu 16.04.

**Comment by serge-rider** [comment]
Can't reproduce.
Please attach debug log (https://github.com/serge-rider/dbeaver/wiki/Log-files).

**Comment by BertHartm** [comment]
Logs attached (with redacted / replaced identifiers)
[_.log.txt](https://github.com/serge-rider/dbeaver/files/642722/_.log.txt)
[dbeaver-debug.log.txt](https://github.com/serge-rider/dbeaver/files/642723/dbeaver-debug.log.txt)

**Comment by serge-rider** [comment]
Debug log is practically empty. And I see just one successful connection log. Error log doesn't contain any connection errors either.

Please provide more details. What "I can no longer connect to those databases" actually means? 
Do you see some error dialog, UI freeze or ..?

**Comment by BertHartm** [comment]
Unfortunately there's not much else I can provide. There's probably not much in the logs because I had recently reinstalled and tried to clean state to work around the issue.

To describe the experience:
I double click on the connection in the database navigator to connect.
The script editor opens in the main pane, but the normal dialogs that happen during connection (where it's fetching data about the db) don't happen, or happen too quickly for me to notice.
If I expand the db in the db navigator I get the loading arrow, which never goes away (though I can collapse the menu): ![image](https://cloud.githubusercontent.com/assets/508811/21153380/3d1f55e4-c138-11e6-8453-1988200081a9.png)
On the right-click context menu for the DB, "connect" is grayed out, and the other items are all active (including "invalidate/reconnect" and "disconnect")
Attempting to run a query from the script editor produces the following error: ![image](https://cloud.githubusercontent.com/assets/508811/21153474/a217be14-c138-11e6-9b9f-7038b08ee124.png)
At this point, "disconnect" does nothing (though it does warn about unsaved changes in the script editor), and attempting to close dbeaver locks out all the controls in the window, and hangs forever.

**Comment by serge-rider** [comment]
Looks like some sort of connection timeout.
- Do you use SSH tunnel?
- Do you save password in connection settings (is "save password locally" checked in settings)?
- What exactly driver version do you use (in connection properties click on "Edit Driver Settings" then expand driver library to see current version)?

To get full debug log:
1. Restart DBeaver. Expand connection in navigator tree.
2. Wait for 20-30 seconds.
3. Exit DBeaver
Now you have full .dbeaver-debug.log. Please post it here again.

**Comment by Kazun3500** [comment]
I had the same problem with 3.8.0 version, with postgres and oracle connections. I returned back to 3.7.8 version, work a little time on it. Some days ago, i upgraded to 3.8.1 and all it worked well.

**Comment by serge-rider** [comment]
BTW make sure you perform version upgrade correctly: you need to remove previous version completely, never extract zip archive over existing DBeaver version - this may lead to unexpected behavior.

**Comment by lBilali** [comment]
I have/had the same issue since updating to 3.8.0 as well.

I use dbeaver as an eclipse extension. 
it was failing under eclipse 4.6 and 4.7.m3 as a clean install.

today while running eclipse from terminal I got the error below
```
*** BUG ***
In pixman_region32_init_rect: Invalid rectangle passed
Set a breakpoint on '_pixman_log_error' to debug
```
a quick search on internet and I found this:
http://stackoverflow.com/questions/36822242/eclipse-doesnt-work-with-ubuntu-16-04

basically adding the 2 lines in `eclipse.ini`
```
--launcher.GTK_version
2
```
hopefully this will help anyone
